pom.xml

```diff
@@ -263,6 +263,13 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-release-plugin</artifactId>
                     <version>${maven-release-plugin.version}</version>
+                    <configuration>
```

Comment:
I'd like to keep configurations in `<build><plugins>` section instead of `<pluginManagement>`

Reply:
if it does not go to a profile, ok